src/components/basic/Avatar/Avatar.tsx

```
@@ -22,30 +28,41 @@ const avatarStyles = cva(
-        `${avatarStyles({ size })} ${
-          isPopoverOpen &&
-          "before:content-[''] before:block before:pointer-events-none before:absolute before:inset-0 before:w-full before:h-full before:bg-overlay-icon"
+        `${avatarStyles({ size })} ${isContentActive && "before:opacity-100"} ${
+          isHoverActive && "hover:before:opacity-100"
         }`,
         className
       )}
+      onClick={handleClick}
     >
+      {isEditable && <EditBadge size={size} />}
       {!!profileImage ? (
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)